h/features.py

```diff
@@ -137,13 +137,17 @@ def remove_old_flags_on_boot(event):
     remove_old_flags()
 
 
+# features_status() provides a JSON-encodeable dictionary of feature-flag
+# keys and current values for a request. Current clients will access it
+# via the /app endpoint. The /app/features endpoint is exposed as a separate
+# service for compatibility with older clients (<= 0.8.6)
 @view_config(route_name='features_status',
              request_method='GET',
              accept='application/json',
              renderer='json',
              http_cache=(0, {'no_store': False}))
 def features_status(request):
-    """Report current feature flag values."""
+    """Returns a dictionary of feature flag keys and their current status"""
```

Comment:
I'm not sure about this change to the docstring. A "dictionary" isn't a thing in JSON.